owlbot.py

```diff
@@ -13,92 +13,114 @@
+        "from google.firestore.v1 import query_pb2  # type: ignore",
+        "from google.cloud.firestore_v1.types import document as document_pb2  # type: ignore\n"
+        "from google.cloud.firestore_v1.types import query as query_pb2 # type: ignore"        
     )
 
-# This library ships clients for 3 different APIs,
-# firestore, firestore_admin and firestore_bundle
-default_version = "v1"
-admin_default_version = "v1"
-bundle_default_version = "v1"
-
-for library in s.get_staging_dirs(default_version):
-    if library.parent.absolute() == 'firestore':
-        s.move(
-            library / f"google/cloud/firestore_{library.name}",
-            f"google/cloud/firestore_{library.name}",
-            excludes=[f"google/cloud/firestore_{library.name}/__init__.py"],
-        )
-
-        s.move(library / f"tests/", f"tests")
-        update_fixup_scripts(library)
-        s.move(library / "scripts")
-
-for library in s.get_staging_dirs(admin_default_version):
-    if library.parent.absolute() == 'admin':
-        s.move(
-            library / f"google/cloud/firestore_admin_{library.name}",
-            f"google/cloud/firestore_admin_{library.name}",
-            excludes=[f"google/cloud/firestore_admin_{library.name}/__init__.py"],
-        )
-        s.move(library / f"tests", f"tests")
-        update_fixup_scripts(library)
-        s.move(library / "scripts")
-
-for library in s.get_staging_dirs(bundle_default_version):
-    if library.parent.absolute() == 'bundle':
-        s.replace(
-            library / "google/cloud/firestore_bundle/types/bundle.py",
-            "from google.firestore.v1 import document_pb2 as gfv_document  # type: ignore\n",
-            "from google.cloud.firestore_v1.types import document as gfv_document\n",
-        )
-
-        s.replace(
-            library / "google/cloud/firestore_bundle/types/bundle.py",
-            "from google.firestore.v1 import query_pb2 as query  # type: ignore\n",
-            "from google.cloud.firestore_v1.types import query\n",
-        )
-
-        s.replace(
-            library / "google/cloud/firestore_bundle/__init__.py",
-            "from .types.bundle import NamedQuery\n",
-            "from .types.bundle import NamedQuery\n\nfrom .bundle import FirestoreBundle\n",
-        )
-
-        s.replace(
-            library / "google/cloud/firestore_bundle/__init__.py",
-            "\'BundledQuery\',",
-            "\"BundledQuery\",\n    \"FirestoreBundle\",",
-        )
-
-        s.move(
-            library / f"google/cloud/bundle",
-            f"google/cloud/firestore_bundle",
-        )
-        s.move(library / f"tests", f"tests")
+    s.replace(
+        library / "google/cloud/bundle/__init__.py",
+        "from .types.bundle import BundleMetadata\n"
+        "from .types.bundle import NamedQuery\n",
+        "from .types.bundle import BundleMetadata\n"
+        "from .types.bundle import NamedQuery\n\nfrom .bundle import FirestoreBundle\n",
```

Comment:
For clarity, either break across lines:
```suggestion
        "from .types.bundle import NamedQuery\n"
        "\n"
        "from .bundle import FirestoreBundle\n",
```
or else use triple-quoted strings.